enigma-coinjoin-client/src/index.js

```diff
@@ -28,10 +28,10 @@ class CoinjoinClient {
 
     async _waitConnectAsync() {
         return new Promise((resolve) => {
-            this.ws.onopen = () => {
+            this.ws.on('open', function open() {
```

Comment:
The other syntax `this.ws.onopen = () =>` does not work for me. Tests are timing out.